packages/mobile/src/tokens/utils.ts

```diff
@@ -30,3 +30,55 @@ export function sortByUsdBalance(token1: TokenBalance, token2: TokenBalance) {
 export function isStablecoin(token: TokenBalance | undefined) {
   return token?.isCoreToken && token.symbol !== 'CELO'
 }
+
+/**
+ * It sorts in 3 categories:
+ * Stable tokens: (cUSD, cEUR, cREAL) (sorted by usd balance)
+ * CELO
+ * Other tokens: sorted by usd balance
+ *
+ * If someone comes with a better name for this, I would appreciate it.
```

Comment:
I really like the name btw, it's very descriptive, I don't think we can do better than that